pkgs/applications/audio/gradio/default.nix

```diff
@@ -51,15 +48,20 @@ in stdenv.mkDerivation rec {
 
     wrapGAppsHook
     desktop-file-utils
-    gsettings-desktop-schemas
-  ] ++ gst_plugins;
```

Comment:
Why remove these? What about radios using ugly codecs?

Reply:
Ok.
Seems I still do not fully understand how buildtime and runtime dependencies work.

Reply:
`wrapGAppsHook` picks up the plug-ins from environment variable set by gstreamer setup hook and wraps the executables with the variable, see https://github.com/NixOS/nixpkgs/pull/43150